test/models/group_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class GroupTest < ActiveSupport::TestCase
+  # test "the truth" do
+  #   assert true
+  # end
+end
```

Comment:
- [ ] Great job so far 👏🏻 However according to the project requirements. The App should have unit tests and integration tests for all the most important components of the RoR application. Kindly make sure all the components have `unit tests` tests ✍🏻